packages/perseus/src/widgets/interactive-graphs/graphs/circle.tsx

```diff
@@ -37,31 +38,101 @@ function CircleGraph(props: CircleGraphProps) {
     const {dispatch, graphState} = props;
     const {center, radiusPoint} = graphState;
 
+    const {strings} = usePerseusI18n();
+    const [radiusPointAriaLive, setRadiusPointAriaLive] = React.useState<
+        "off" | "polite"
```

Comment:
I'd recommend using the aria live type in
packages/perseus/src/widgets/interactive-graphs/types.ts